ndex2/client.py

```diff
@@ -809,6 +1115,23 @@ def set_read_only(self, network_id, value):
     def set_network_properties(self, network_id, network_properties):
         """
         Sets network properties
+	
+	.. code-block:: python
+
+		import ndex2.client as nc
+		import ndex2
+		import networkx as nx
+		import pandas as pd
+		import os
+
+		network_id = 'INSERT NETWORK UUID HERE'
+
+		#the preoperties are set as a list of value pairs
+		network_properties = [("isReadOnly",True),
+				      ("visibility","PUBLIC"),
+				      ("isShowcase",True)]
+
+		my_ndex.set_network_properties(network_id, network_properties)
```

Comment:
Please add line showing how `my_ndex` is created. The only import needed is the one to import the Ndex2 client